kdlib/source/kdlib.vcxproj

```diff
@@ -455,24 +455,34 @@
-    <Import Project="$(SolutionDir)\packages\boost_system-src.1.67.0.0\build\boost_system-src.targets" Condition="Exists('$(SolutionDir)\packages\boost_system-src.1.67.0.0\build\boost_system-src.targets')" />
-    <Import Project="$(SolutionDir)\packages\boost_thread-src.1.67.0.0\build\boost_thread-src.targets" Condition="Exists('$(SolutionDir)\packages\boost_thread-src.1.67.0.0\build\boost_thread-src.targets')" />
+    <Import Project="$(SolutionDir)\packages\boost.1.72.0.0\build\boost.targets" Condition="Exists('$(SolutionDir)\packages\boost.1.72.0.0\build\boost.targets')" />
+    <Import Project="$(SolutionDir)\packages\boost_atomic-src.1.72.0.0\build\boost_atomic-src.targets" Condition="Exists('$(SolutionDir)\packages\boost_atomic-src.1.72.0.0\build\boost_atomic-src.targets')" />
+    <Import Project="$(SolutionDir)\packages\boost_chrono-src.1.72.0.0\build\boost_chrono-src.targets" Condition="Exists('$(SolutionDir)\packages\boost_chrono-src.1.72.0.0\build\boost_chrono-src.targets')" />
+    <Import Project="$(SolutionDir)\packages\boost_date_time-src.1.72.0.0\build\boost_date_time-src.targets" Condition="Exists('$(SolutionDir)\packages\boost_date_time-src.1.72.0.0\build\boost_date_time-src.targets')" />
+    <Import Project="$(SolutionDir)\packages\boost_regex-src.1.72.0.0\build\boost_regex-src.targets" Condition="Exists('$(SolutionDir)\packages\boost_regex-src.1.72.0.0\build\boost_regex-src.targets')" />
+    <Import Project="$(SolutionDir)\packages\boost_system-src.1.72.0.0\build\boost_system-src.targets" Condition="Exists('$(SolutionDir)\packages\boost_system-src.1.72.0.0\build\boost_system-src.targets')" />
+    <Import Project="$(SolutionDir)\packages\boost_thread-src.1.72.0.0\build\boost_thread-src.targets" Condition="Exists('$(SolutionDir)\packages\boost_thread-src.1.72.0.0\build\boost_thread-src.targets')" />
+    <Import Project="..\..\..\packages\boost.1.72.0.0\build\boost.targets" Condition="Exists('..\..\..\packages\boost.1.72.0.0\build\boost.targets')" />
+    <Import Project="..\..\..\packages\boost_chrono-src.1.72.0.0\build\boost_chrono-src.targets" Condition="Exists('..\..\..\packages\boost_chrono-src.1.72.0.0\build\boost_chrono-src.targets')" />
+    <Import Project="..\..\..\packages\boost_date_time-src.1.72.0.0\build\boost_date_time-src.targets" Condition="Exists('..\..\..\packages\boost_date_time-src.1.72.0.0\build\boost_date_time-src.targets')" />
+    <Import Project="..\..\..\packages\boost_regex-src.1.72.0.0\build\boost_regex-src.targets" Condition="Exists('..\..\..\packages\boost_regex-src.1.72.0.0\build\boost_regex-src.targets')" />
+    <Import Project="..\..\..\packages\boost_system-src.1.72.0.0\build\boost_system-src.targets" Condition="Exists('..\..\..\packages\boost_system-src.1.72.0.0\build\boost_system-src.targets')" />

```

Comment:
These lines seem to be referencing a path outside the build directory, right? Are you putting the boost library somewhere outside the pykd directory?

If not, can you change these "`..\..\..\`" to "`$(SolutionDir)`" and confirm that it'll still build?